### PR TITLE
add ocm label for hub backup and recovery

### PR DESCRIPTION
**Pull request by BenamarMk**

### CODE REVIEW

**Comment by BenamarMk** [comment]
This PR is to add a label to drpc in order to comply with https://issues.redhat.com/browse/ACM-1001

**Review by ShyamsundarR — APPROVED** [review]
LGTM! Waiting on other PRs in queue before merge (and possible conflict resolution if that happens)

**Pull request merged**
